server/main.py

```diff
@@ -153,6 +154,9 @@ async def index(request):
         job.cpu(cpu)
     if memory:
         job.memory(memory)
+    if preemptible is not None:
```

Comment:
Should we default to True here if `None` to be consistent with the CLI description?

Reply:
Good idea! I've added a default value when fetching the value from the dictionary.